lib/utils.js

```diff
@@ -35,6 +36,7 @@ exports.inherits = util.inherits;
  * @return {string}
  */
 exports.escape = function (html) {
+  html = html.toString().replaceAll(RESTRICTED_CHAR, "�");
```

Comment:
🤔 this looks like a duplicate of #4527. @malloryavvir do you have thoughts on https://github.com/mochajs/mocha/pull/4527/files#r1511131215 ?